### ISSUE TITLE
Problems of dependencies versions in environment.yml

### ISSUE THREAD

**Issue by tegg89** [rll/rllab#182]
Hi, I am trying to follow guide to install rllab.
However, versions of some packages in environment.yml do not match well.
For example, Theano, Lasagne, Plotly, and Chainer do not install completely 
(maybe they already have merged).
Is it okay to install those libraries separately? 
IMO, the latest versions of libraries and those depended on the rllab would not match, causing errors.

**Comment by tegg89** [comment]
I had figured this out. 
Instead, I just installed the latest version of frameworks and required dependencies separately.
Afterward, the example code could run completely.

**Comment by 0fork** [comment]
Yeah, nothing works with this library when it comes to setup. Fails on both Ubuntu and macOS.

**Comment by christopburns** [comment]
I had to go to https://github.com/Lasagne/Lasagne and follow the installation instructions to get it working in my environment.
